src/app/modules/main/communities/module.nim

```diff
@@ -92,16 +94,21 @@ method viewDidLoad*(self: Module) =
   self.delegate.communitiesModuleDidLoad()
 
 method onActivated*(self: Module) =
+  if self.curatedCommunitiesLoaded:
+    return
```

Comment:
I guess the question here is: do we really want that? What if curated communities have changed in the meantime? Or do we update curated communities at runtime in a different way?

Reply:
I don't think we do at the moment, but the curated community feature is super broken at the moment anyway. Patryk is implementing the new one as we speak. 

Ideally, we would update the model using signals once we load them, so we don't have the re-load everything when we go back to the section, but we can change that if we prefer unloading the section on leave. We'll see when we get there.